app/modules/users/model.js

```diff
@@ -1,23 +1,14 @@
 // @flow
 
 import type { Identifier } from 'types/model';
-import generateRandomString from 'lib/generate-random-string';
 
-// #TODO optimal id length / generation method?
-const ID_LENGTH = 10;
-
-export type User = {
+export type UserType = {
```

Comment:
Why the change? In other modules object types don't explicitly end in `Type` either, see for example `Topic` or `ContentItem`.

Reply:
My two cents: I named it `FeedItemsType` in the feed-items module. Maybe that's where the idea came from. I think it's less ambiguous with the -Type suffix.